### PR TITLE
Add help message in celery command

### PR DESCRIPTION
**Pull request by zeroam**
Celery command is available just for CeleryExecutor.
The others cannot recognize.
It needs to show help message for celery command even if
the executor is not CeleryExecutor.

---
Issue link: WILL BE INSERTED BY [boring-cyborg](https://github.com/kaxil/boring-cyborg)

Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Comment by mik-laj** [comment]
Hi. 
We recently had discussions about whether it is a good idea to hide commands in CLI. We decided that we prefer it to be visible, but with the appropriate warning.
What do you think to enable showing Celery command for everyone and add appropriate warnings?
https://github.com/apache/airflow/pull/7070

File: airflow/cli/cli_parser.py [hunk 1]
```diff
@@ -55,6 +56,28 @@ def command(*args, **kwargs):
...
 
 
+class DefaultHelpParser(argparse.ArgumentParser):
+    """CustomParser to display help message"""
+    custom_msg = {'celery': 'celery command cab be used in CeleryExcutor'}
+
+    def _check_value(self, action, value):
+        """Override _check_value and check conditionally added command"""
+        if action.choices is not None and value not in action.choices:
+            args = {'value': value,
```
Comment (mik-laj) [inline airflow/cli/cli_parser.py]:
```suggestion
    if value in self.custom_msg:
        msg = self.custom_msg[value]
    else:
        args = {
            'value': value,
            'choices': ', '.join(map(repr, action.choices))
        }
        msg = gettext('invalid choice: %(value)r (choose from %(choices)s)') % args
```
It is not necessary to generate a generic message in all cases.  If we have a special message, we display it. When we don't have it, we create it.

**Comment by zeroam** [comment]
> Hi.
We recently had discussions about whether it is a good idea to hide commands in CLI. We decided that we prefer it to be visible, but with the appropriate warning.
What do you think to enable showing Celery command for everyone and add appropriate warnings?
#7070

It was difficult for me to figure out the reason why celery command is not visible, 
I agree with this opinion https://github.com/apache/airflow/pull/7070#pullrequestreview-338798365

and, I think, if celery command is showed to everyone and it's not available, (ex. executor is not CeleryExecutor, or DB is sqlite) appropriate message is set executor to CeleryExecutor and change database which is not sqlite.

File: airflow/cli/cli_parser.py (hunk 1 above)
Comment (zeroam) [reply airflow/cli/cli_parser.py]: I agree

**Comment by zeroam** [comment]
I will propose another PR to set visible celery command

**Pull request closed without merging**